WNPRC_Purchasing/src/client/components/RequestOrderPanelInputs.tsx

```diff
@@ -86,14 +87,15 @@ export const AccountOtherInput: FC<InputProps> = memo(props => {
 });
 
 interface VendorInputProps {
-    onChange: (colName, value) => void;
+    onChange?: (colName, value) => void;
     hasError?: boolean;
     model: RequestOrderModel;
-    onModelChange: (model: RequestOrderModel) => void;
+    onModelChange?: (model: RequestOrderModel) => void;
```

Comment:
Same thing here, if making onChange and onModelChange optional you might want to optionally call them below so you don't get NPE

Reply:
done